libs/wai-utilities/src/Network/Wai/Utilities/Server.hs

```diff
@@ -361,6 +361,7 @@ onError g m r k e = liftIO $ do
   when (code >= 500) $
     either Prm.incCounter (counterIncr (path "net.errors")) `mapM_` m
   flushRequestBody r
+  Log.flush g
```

Comment:
Why is this necessary?

Reply:
It's not, it sneaked in there somehow... good catch!